integration/compatibility/generate_alias.py

```
@@ -0,0 +1,81 @@
+import pytest
+from .data_sets import *
```

Ruff (0.15.18)

[error] `from .data_sets import *` used; unable to detect undefined names

(F403)